pkg/pillar/pubsub/change.go

```diff
@@ -23,3 +25,19 @@ type Change struct {
 	// Value the value of the affected item, if any
 	Value []byte
 }
+
+// String returns string representation of Change object
+func (change Change) String() string {
+	operation := ""
+	switch change.Operation {
+	case Restart:
+		operation = "Restart"
+	case Create:
+		operation = "Create"
+	case Delete:
+		operation = "Delete"
+	case Modify:
+		operation = "Modify"
+	}
+	return fmt.Sprintf("operation %s key %s val %s", operation, change.Key, string(change.Value))
```

Comment:
We shouldn't log the value because the value can contain things like user credentials.

Printing the operation as a String would be useful though.
Defining an func (op Operation)String() function would do that.